helix-core/src/main/java/org/apache/helix/messaging/handling/HelixTaskExecutor.java

```diff
@@ -488,11 +488,17 @@ public boolean scheduleTask(MessageTask task) {
               "Message handling task already sheduled for " + taskId, manager);
         }
       }
-    } catch (Exception e) {
-      LOG.error("Error while executing task. " + message, e);
-      _statusUpdateUtil
-          .logError(message, HelixTaskExecutor.class, e, "Error while executing task " + e,
-              manager);
+    } catch (Throwable e) {
+      String errorMessage = "Error while executing task. " + message;
+      LOG.error(errorMessage, e);
```

Comment:
Please consider using standard log message consolidation. It will safe new string object creation. 
Log.error("xxxxx {} " , var);

Reply:
thanks @xyuanlu , fixed this.